### PR TITLE
fix(docs): site layout

### PR DESCRIPTION
**Pull request by rachelbt**

### CODE REVIEW

**Comment by jshenkman** [comment]
Looks great in both schemes !! Thanks @rachelbt for the fix :)

File: apps/docs/_layouts/base.njk [hunk 1]
```diff
@@ -49,78 +49,36 @@
...
-          </defs>
-        </svg>
-
-        <vwc-text font-face="body-2" tight>Web Components</vwc-text>
-      </div>
-    </vwc-header>
 
+     <vwc-header id="header-main" slot="app-content">
+      <a class="logo-wrapper" href="/" alt="Vivid Web Components">
+       <img src="/vivid-logo.svg" style="" alt="Vivid Logo" width="40">
```
Comment (yinonov) [inline apps/docs/_layouts/base.njk:56]: Works in both dark / light?

File: apps/docs/assets/styles/main.css [hunk 2]
```diff
@@ -53,39 +53,41 @@ vwc-side-drawer::part(base) {
...
-  inline-size: auto;
-  inset-inline: var(--header-inset-inline-start) 0;
-}
-
-vwc-header#header-main > main {
-  margin-top: var(--_header-block-size);
-}
-
-vwc-header#header-aside::part(base) {
   position: sticky;
```
Comment (yinonov) [inline apps/docs/assets/styles/main.css:69]: This bounces with elements in the web kit bouncing scroll behavior

**Review by yinonov — CHANGES_REQUESTED** [review]

**Review by yinonov — CHANGES_REQUESTED** [review]
Looks a bit damaged in mobile. Suggesting to focus and merge #538 first
image

**Comment by rachelbt** [comment]
> Looks a bit damaged in mobile. Suggesting to focus and merge #538 first image

didn't happen to me. 
There are no fonts in your print screen - maybe it wasn't fully loaded?
It looks OK once the drawer is closed :)
image

File: apps/docs/_layouts/base.njk (hunk 1 above)
Comment (rachelbt) [reply apps/docs/_layouts/base.njk]: yes. Its only the image and not the text

File: apps/docs/assets/styles/main.css (hunk 2 above)
Comment (rachelbt) [reply apps/docs/assets/styles/main.css:69]: > This bounces with elements in the web kit bouncing scroll behavior image

didn't happen to me. mind that theres no fonts in your printscreen. Maybe its not fully loaded.

**Comment by rachelbt** [comment]
@yinonov - ready for re-check.
One thing though - there's a horizontal scroll in mobile because of long headline combined with too big font.
When theres scroll than the side -drawer and the header breaks.
We have a typography PR so I'm leaving it for now.

**Comment by rachelbt** [comment]
@jshenkman 
for Visual QA:
- now the header is starched above the side-drawer
- in main page: added the grey in the first part
- in all component page - added bg to the component name. (didn't add the image - leaving it to another PR).
- please check both desktop & mobile
- there might be some breaking in mobile due to the typography and lone names of the components member (e.g. Accessibility).
Thanks

**Review by yinonov — APPROVED** [review]

**Pull request merged**